frontend/src/metabase/setup/components/Setup.jsx

```diff
@@ -212,6 +213,11 @@ export default class Setup extends Component {
                 backgroundColor: color("white"),
               }}
             />
+            <DriverWarning
+              engine={selectedDatabaseEngine}
+              ml={26}
+              data-testid="database-setup-driver-warning"
+            />
```

Comment:
Are we sure we want to display `DriverWarning` under the `AddDatabaseHelpCard` here? `AddDatabaseHelpCard` is always displayed, but the `DriverWarning` can be shown only for specific drivers, so if you change drivers on the setup, the cards would jump

Reply:
True.  I guess I misunderstood your comment from above:

> The DriverWarning is invisible now because of the page layout. You can move it under the AddDatabaseHelpCardHolder and remove the left margin